### PR TITLE
LPS-189820 Asset categories ids are duplicated 

### PR DESCRIPTION
**Pull request by ambrinchaudhary**
Issue https://liferay.atlassian.net/browse/LPS-189820. 

Please let me know if this approach is not the correct. Thanks!

### CODE REVIEW

File: modules/apps/asset/asset-taglib/src/main/resources/META-INF/resources/asset_categories_selector/AssetCategoriesSelectorTag.es.js [hunk 1]
```diff
@@ -104,6 +104,7 @@ export default function (props) {
 
 					<AssetCategoriesSelectorTag
 						{...props}
+						id={publicVocabulariesId}
```
Comment (veroglez) [inline modules/apps/asset/asset-taglib/src/main/resources/META-INF/resources/asset_categories_selector/AssetCategoriesSelectorTag.es.js:107]: These `ids` are already being used in the group tag here https://github.com/liferay-echo/liferay-portal/blob/618f69e623f180911865419a74013c230c90b9fd/modules/apps/asset/asset-taglib/src/main/resources/META-INF/resources/asset_categories_selector/AssetCategoriesSelectorTag.es.js#L92C1-L93C1. 

We can add something like `{publicVocabulariesId}_categories` to the id to make it different. The same for `internalVocabulariesId`

**Comment by veroglez** [comment]
Hi @ambrinchaudhary, Thanks for sending the fix! I've left you a comment.

File: modules/apps/asset/asset-taglib/src/main/resources/META-INF/resources/asset_categories_selector/AssetCategoriesSelectorTag.es.js (hunk 1 above)
Comment (ambrinchaudhary) [reply modules/apps/asset/asset-taglib/src/main/resources/META-INF/resources/asset_categories_selector/AssetCategoriesSelectorTag.es.js]: Thanks @veroglez, you are right, I just changed the current commit with your suggestion

**Comment by veroglez** [comment]
ci:test:sf

**Review by veroglez — APPROVED** [review]

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**